manager/src/main/java/org/openremote/manager/dashboard/DashboardResourceImpl.java

```diff
@@ -178,7 +190,7 @@ protected DashboardQuery createDashboardQuery(String realm) {
         assetAccess.add(DashboardQuery.AssetAccess.REALM);
 
         // Adjust query object based on user roles/permissions
-        if (isAuthenticated()) {
+        if (isSuperUser() || (isAuthenticated() && realm.equals(getAuthenticatedRealmName()))) {
```

Comment:
I think this logic should be reversed and return a `FORBIDDEN` error if authenticated non super user is requesting another realm then return the 403.

Not sure just accepting it and giving them public dashboards of that realm is good. If they're doing something bad then let them know.

Reply:
Good suggestion! I've updated it in the newest changes. 👍